msg/de_DE.utf8/cubrid.msg

```diff
@@ -1336,8 +1336,8 @@ $ LOADDB
 1265 Error connecting with the page server at %s.
 1266 The high availability system is not compatible with the page server type.
 1267 Cannot copy volume %s to %s. Error: %s.
-1268 Error reserved for scalability development.
-1269 Error reserved for scalability development.
+1268 Inconsistent setting: remote storage is specified, but no page server found in configuration.
+1269 Inconsistent setting: remote storage is specified, but no connection to page servers is possible.
```

Comment:
The remote/local storage is a temporary behavior, in the end it will be always remote. These errors will have to be changed then, or we could drop the mention about remote storage.

```suggestion
1268 At least one valid page server host is necessary, but no page server found in configuration.
1269 At least one valid page server host is necessary, but no page server could be connected.
```

Reply:
Done